01_fib/ailyntang/fib.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"fmt"
+	"io"
+	"os"
+)
+
+var out io.Writer = os.Stdout
+
+func fib(n int) {
+	if n < 1 || n > 92 {
+		fmt.Fprintln(out, "fib.go only handles integers between 1 to 92 inclusive")
+		return
+	}
+
+	series := genFib(n)
+
+	for _, num := range series {
+		fmt.Fprintln(out, num)
+	}
+}
+
+func genFib(n int) []int64 {
+	series := make([]int64, n)
+	series[0] = 1
+
+	if n == 1 {
+		return series
+	}
+
+	series[1] = 1
+	for i := int(2); i < n; i++ {
```

Comment:
The `int(...)` cast here is unnecessary - just `for i := 2; i < n; i++ {` will do.

Reply:
Thanks, I'll update it